frontend/src/components/Listing/listing.jsx

```diff
@@ -125,16 +125,18 @@ class Listing extends Component {
                           alt="apartment"
                         />
                     }
-                      
                   </span>
 
                   <div className = "listingTextAndIcon">
-                    <span className = "listingHeader">
-                    
+                    <div className = "listingHeader">
```

Comment:
This is from an attempt to try to introduce a newline between the title and the icons (so there's more space for the title).